src/frontend-scripts/components/section-main/CardFlinger.jsx

```diff
@@ -77,13 +80,9 @@ class CardFlinger extends React.Component {
 
 			if (userInfo.gameSettings && userInfo.gameSettings.disableHelpMessages) {
 				return;
-			}
-
-			if (status === 'Fascists win the game.' || status === 'Liberals win the game.') {
+			} else if (status === 'Fascists win the game.' || status === 'Liberals win the game.') {

```

Comment:
None of these (83-124) need to be `else if`s, and they theoretically shouldn't be considering they're all `return`ers.